### PR TITLE
Docs: Fix img link for alert notification template

### PR DESCRIPTION
**Pull request by wbrowne**
Corresponding update to image file name: https://github.com/grafana/website/pull/3214

### CODE REVIEW

**Comment by oddlittlebird** [comment]
The images for the What's New doc should be in the v74 folder so that they are not accidentally updated. Documentation images should be in subject folders, such as `alerting`, and should follow the [image file naming conventions](https://github.com/grafana/grafana/blob/master/contribute/style-guides/documentation-style-guide.md#file-naming-conventions).

File: docs/sources/whatsnew/whats-new-in-v7-4.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ The main use case is for [multi-dimensional](https://grafana.com/docs/grafana/la
 
 You can now provide detailed information to alert notification recipients by injecting alert query data into an alert notification. Labels that exist from the evaluation of the alert query can be used in the alert rule name and in the alert notification message fields. The alert label data is injected into the notification fields when the alert is in the alerting state. When there are multiple unique values for the same label, the values are comma-separated.
 
-![Variable support in alert notifications](/img/docs/v74/alert-variable-notifications.png)
+![Variable support in alert notifications](/img/docs/alerting/notification_template.png)
```
Comment (achatterjee-grafana) [inline docs/sources/whatsnew/whats-new-in-v7-4.md:84]: @wbrowne, from the [Documentation style guide](https://github.com/grafana/grafana/blob/master/contribute/style-guides/documentation-style-guide.md):

Image file names should be descriptive and unique. Also, add the software version number that the image applies to or the screenshot was taken in. Example: share-dashboard-link-7-3.png.

If you can append the version number, that will be great!

**Comment by wbrowne** [comment]
@oddlittlebird So even if the image exists in the docs already, we should duplicate?

**Comment by oddlittlebird** [comment]
Correct. Because one image should never be updated, the one in the What's New. It documents what happened in the release. The other image will change based on the documentation content.

**Comment by achatterjee-grafana** [comment]
@oddlittlebird, how about we keep the image in alerting and relref it in 7.4 what’s new. 

If you update the image in the doc topic, in later releases, say v7.5, you will anyway create an image with 7-5, correct? If we do this, we can bypass image duplication. They do take up a lot of space. Thoughts?

**Comment by oddlittlebird** [comment]
That's a good point. We can do that, provided the file name is updated.

**Review by achatterjee-grafana — APPROVED** [review]
LGTM

**Pull request merged**